src/types/aliases.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2023  David Kellner
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import {LanguageT} from './language';
+import {LazyLoaded} from './utils';
+
+
+export type AliasT = {
+	id: number,
+	name: string,
+	sortName: string,
+	languageId?: number,
```

Comment:
```suggestion
	languageId: number | null,
```
I haven't checked yet whether it makes more sense to mark these columns as nullable instead of optional.

Reply:
Ok, I had a look again at the JSON samples which I had extracted from the React initial page properties a while ago. I did not find an alias without language, but all occurrences of the columns which I marked as optional in this PR were always present and filled with `null` in my samples if they have no other value.

No idea why I initially decided to mark them as optional, but I will change these to be nullable.